cartography/data/jobs/cleanup/aws_post_ingestion_dns_cleanup.json

```diff
@@ -1,14 +1,40 @@
 {
   "statements": [
     {
-      "query": "MATCH (n:DNSRecord) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+      "query": "MATCH (n:AWSDNSRecord) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
This cleanup job runs at the [end of a single AWS account's sync](https://github.com/lyft/cartography/blob/c39a3bbfc1e8a7f8a6a04658467508b644db1c89/cartography/intel/aws/__init__.py#L48).  Other sync jobs in the future such as GCP might use the generic `:DNSRecord` label, so we should not delete all stale `:DNSRecords`.  Rather, we should only delete `:AWSDNSRecords`.